packages/edit-post/src/components/layout/index.js

```diff
@@ -293,6 +294,7 @@ function Layout( { initialPost } ) {
 			) }
 			<PostEditorMoreMenu />
 			<BackButton initialPost={ initialPost } />
+			<EditorSnackbars />
```

Comment:
The snackbars need to be rendered only once in the site editor (or the post editor) so I left this responsibility out of the EditorCanvas component for now.